### PR TITLE
Vendor fixes

### PR DESCRIPTION
**Pull request by GermanBluefox**
Formatting
Change type of object "system.meta.uuid" to "config"
Added description of nonEdit attribute

### CODE REVIEW

File: packages/cli/src/lib/setup/setupVendor.ts [hunk 1]
```diff
@@ -4,6 +4,26 @@ import fs from 'fs-extra';
 import deepClone from 'deep-clone';
 import { isDeepStrictEqual } from 'util';
 
+/*
```
Comment (foxriver76) [inline packages/cli/src/lib/setup/setupVendor.ts:7]: I would add these informations on type level and also allow the property, I can push into the PR  branch if okay for you.

File: packages/common/src/lib/common/tools.ts [hunk 2]
```diff
@@ -466,7 +470,7 @@ function updateUuid(newUuid: string, _objects: any, callback: (uuid?: string) =>
         _objects.setObject(
             'system.meta.uuid',
             {
-                type: 'meta',
```
Comment (foxriver76) [inline packages/common/src/lib/common/tools.ts:469]: For me personally it is a bit weird, because of the id itself containing meta^^

**Comment by foxriver76** [comment]
Would be great to have some feedback, which changes of the PR are important and why @GermanBluefox 

So if the change from `meta` to `config` is not necessary, I would prefer to keep it as `meta` for now. And then the only question remains is about the change for vendor cli without providing a path to vendor file. I would then also like to document it in the yargs help, so you get auto completion and help for this option

**Comment by Apollon77** [comment]
@GermanBluefox @foxriver76 How continue here?

File: packages/common/src/lib/common/tools.ts (hunk 2 above)
Comment (GermanBluefox) [reply packages/common/src/lib/common/tools.ts:469]: It is historical

**Comment by GermanBluefox** [comment]
> change for vendor cli without providing a path to vendor file

There is a default path for vendor file, so it is not necessary to set the path explicitly.

**Comment by GermanBluefox** [comment]
@foxriver76  @Apollon77 
What is now with this PR?

**Comment by foxriver76** [comment]
> @foxriver76 @Apollon77 What is now with this PR?

We have created another PR with only the relevant changes, as we have included the types for vendor specific operations in the meantime, also the other PR has auto completion for the vendor operation on CLI. 

We will check as soon as @Apollon77 is back. Long story short: The changes will go into new controller v5 for sure.

**Pull request closed without merging**